### DISCUSSION TITLE
menurole.go appears to have the HideRole available but it's commented out

### DISCUSSION THREAD

**Discussion by keyle** [wailsapp/wails#1472] [Q&A] [answered]
Hi,

I'd like the application to respond to `cmd+h` on macOS which is common and I see in the source that it appears to be there but commented out.

ref. https://github.com/wailsapp/wails/blob/master/v2/pkg/menu/menuroles.go

They appear to be missing (or not implemented yet) here:
https://github.com/wailsapp/wails/blob/master/v2/internal/frontend/desktop/darwin/Role.h

Am I looking int the wrong location or is there a way for me to implement that? If it's fairly trivial I'm happy to help.

**Comment by leaanthony** [comment, marked as answer]
Not implemented yet. You can bind the keys manually mind...

If you want to implement the role then that'd be great. Roles should work cross platform hence not being completed yet.

**Reply by keyle** [reply]
I'm afraid I can't help for the other platforms. Any example of how I can do it manually?

**Reply by leaanthony** [reply]
https://wails.io/docs/reference/menus

**Reply by keyle** [reply]
Ok, no worries, yes I've read that doc, and made my own menus already. I was looking to use the default OSX one, that includes "Hide _ApplicationName_".

I will do it "by hand" instead, calling `WindowHide` and map to `cmd+h`. Thanks for the replies.

**Reply by leaanthony** [reply]
The roles are based on electron's idea of roles so if it's Mac only, feel free to add it 👍
